helm2yaml/check_repo.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/local/bin/python3
+from applib.helm import Helm
+from applib.repo import Repo, RepoType
+import sys, os, time, getopt, yaml
+from common import *
+
+TEMPDIR="tmp"
+
+def printhelp():
+  print(('''
+check repositories from the given manifest
+
+Usage: {0} -m manifest [-t target-repository-to-compare ][-v]
```

Comment:
```suggestion
Usage: {0} -m <manifest> [-r <target repository to check> ] [-e <execption list>] [-t <type> (default: chart)] [-v]
```
아래 설명된 내용과 안맞거나 누락된 부분 추가해서 보여주는 것이 좋겠습니다.

Reply:
-m <manifest> 도 -m <HelmRelease manifest> 정도로 바꾸는 것이 좋겠습니다.